Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -273,6 +301,13 @@
   - type: Butcherable
     meat: FoodMeat
     pieces: 5
+  - type: Grammar
+    attributes:
+      gender: female # Here because of UdderComponent
```

Comment:
this is otis from back at the barnyard erasure

Reply:
Known. Check the timestamp to my buddy the other day. 😎 

![image](https://user-images.githubusercontent.com/97707302/153336842-a982a82c-aa9e-4816-afa7-e0881fca3bba.png)

Also back in my day 👴 it was just called Barnyard and it was the prequel movie detailing Tighten from Megamind's backstory.

Reply:
(This one's resolved but if I click the button it'll hide all these great jokes)